src/lib.rs

```diff
@@ -47,26 +61,25 @@ pub extern "C" fn game_loop() {
     cheat_menu::apply_cheats();
     let d_down = controller::DPAD_DOWN.is_pressed();
     let rt_down = controller::R.is_down();
+    let console = Console::get();
```

Comment:
This is potentially dangerous. If this is a mutable reference, we can't alias it in the other menus without leaning towards undefined behavior.

Reply:
For now I'll leave this in, as this isn't the only thing broken in our Wind Waker code like this. We need a whole lot more Cell / UnsafeCell usage. We'll do this in further cleanup passes after this PR.